mappings/net/minecraft/server/network/ServerPlayerEntity.mapping

```diff
@@ -8,12 +8,18 @@ CLASS ut net/minecraft/server/network/ServerPlayerEntity
 	FIELD cb clientLanguage Ljava/lang/String;
 	FIELD cd advancementManager Lqm;
 	FIELD cq clientChatVisibility Larj$b;
+	FIELD cs lastActionTime J
+	FIELD cu inTeleportationState Z
 	FIELD cv seenCredits Z
 	FIELD cw recipeBook Lxw;
 	FIELD d interactionManager Luu;
+	FIELD g seesCredits Z
```

Comment:
Maybe call it `isNotInAnyWorld`? It's used to skip certain parts of the movement code when the player is connected but not in any world. Seeing the vanilla credits screen is an example of this, but a mod may want to add their own "not in any world" state.

Reply:
Good idea.